tests/configure_user/configure_user/tasks/configure_user.yml

```diff
@@ -0,0 +1,52 @@
+---
+- debug: msg="START configure_user function on connection={{ ansible_connection }}"
+
+- name: ensure netconf is enabled
+  junos_netconf:
+    netconf_port: 22
```

Comment:
Please change this to `netconf_port: "{{ netconf_port }}"

Reply:
Done.